src/desktop/analytics/criteo.js

```diff
@@ -3,18 +3,19 @@
 //
 window.criteo_q = window.criteo_q || []
 var pathSplit = location.pathname.split("/")
+const pageType = window.sd.PAGE_TYPE || pathSplit[1]
```

Comment:
The updates here are to allow an override based on `PAGE_TYPE` and then fall back to the URL, like we do elsewhere.